### DISCUSSION TITLE
Does this work in M365 GCC High?

### DISCUSSION THREAD

**Discussion by mike16329** [silverhack/monkey365#74] [Q&A]
Does this work in M365 GCC High?

**Comment by silverhack** [comment]
Hey @mike16329, 

Although not fully tested with Gov instances, the answer is yes. Monkey365 supports Gov instances through Environment parameter. You can find more information here:

[https://silverhack.github.io/monkey365/configuration/general-options/](https://silverhack.github.io/monkey365/configuration/general-options/)
